src/test/reporters/json.ts

```diff
@@ -200,9 +201,20 @@ class JSONReporter implements Reporter {
   }
 
   private _serializeTestSpec(test: TestCase): JSONReportSpec {
+
+    const fixedTags: string[] = [];
+    const tagMatches: RegExpMatchArray| null = test.title.match(/@[\S]+/g);
+
+    if (tagMatches) {
```

Comment:
```js
const tags = (test.title.match(/@[\S]+/g) || []).map(t => t.substring(1));
```

Reply:
Going that route didn't even require assignment up front. Could just set tags inline during the return statement. Got an updated commit in. Thanks for the help!